forest/satellite.py

```diff
@@ -98,10 +103,16 @@ def find_index(times, time, length):
         valid_times = np.ma.array(times, mask=~inside)
         if valid_times.mask.all():
             msg = "{}: not found".format(time)
+            print(msg)
             raise IndexNotFound(msg)
         return np.ma.argmax(valid_times)
 
     @staticmethod
     def parse_date(path):
         groups = re.search(r"([0-9]{8})\.nc", path)
-        return dt.datetime.strptime(groups[1], "%Y%m%d")
+        if groups is None:
+            # *20191211T0000Z.nc
```

Comment:
stray comment.

Reply:
This comment could be extended to explain itself